project.json

```diff
@@ -0,0 +1,32 @@
+{
+  "targets": {
+    "prisma-studio": {
+      "executor": "nx:run-commands",
+      "options": {
+        "cwd": "./packages/server/",
+        "command": "prisma studio"
+      }
+    },
+    "prisma-seed": {
+      "executor": "nx:run-commands",
+      "options": {
+        "cwd": "./packages/server/",
+        "command": "prisma db seed"
+      }
+    },
+    "prisma-migrate": {
+      "executor": "nx:run-commands",
+      "options": {
+        "cwd": "./packages/server/",
+        "command": "prisma migrate dev"
```

Comment:
Since you know this is an interactive command asking for the name of migration, how it will work then ??

Reply:
We can use `--name` flag. This will make generate command non interactive

Reply:
Nx supports argument forwarding

Reply:
Yes i see